surround/tests/cli/project_cli/Test_surround_init_lint.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+import subprocess
+import shutil
+import os
+class MyTestCase(unittest.TestCase):
+#validating the surround init command line functionality
+#verfiying the output after the surround init command
+    def check_project_name(self):
+        self.test_surround_init()#for checking the Name of project input
```

Comment:
This is not what we were looking for
We were saying create something like the following:
```
def check_project_name(self, project_name, is_valid):
    # TODO: Create project code here
    process = ...

    if is_valid:
        self.assertIn("info: project created at", process.stdout)
    else:
        self.assertIn("surround: error: ". process.stdout)

def test_project_name(self):
    self.check_project_name("project_name", is_valid=True)
    self.check_project_name("PROJECT_NAME", is_valid=False)
    self.check_project_name("12343", is_valid=False)
    self.check_project_name("project_name_123", is_valid=False)
```